Sleep Tracker/SleepTracker/View/SleepSetupView/SleepSetupView.swift

```diff
@@ -2,28 +2,41 @@ import SwiftUI
 
 /// A view shows the options that the user can set before go to bed.
```

Comment:
As we discussed in Slack it's not a good description. Maybe it would be better to write it this way?

```suggestion
// A view where the user can set the alarm and choose specific time when it goes off.
```

Reply:
Agree, missed the point, thanks!